requirements.txt

```diff
@@ -0,0 +1,13 @@
+Django
+Jinja2
+MarkupSafe
+PyYAML
+Pygments
+Sphinx
+argparse
+docutils
+mock
+pep8
+psycopg2
+requests
+wsgiref
```

Comment:
Are we still storing a copy of this in dockerfiles? That's not a very good idea, if we can remove the need for a copy, that would be best. If that's complicated, I am ok merging this for all of the other project setup contains and fixing the copy in a new issue.